scripts/cfn

```diff
@@ -131,7 +134,11 @@ if __name__ == "__main__":
 
         if values.s3bucket:
             # Upload to S3 and create the stack
-            s3conn = boto.s3.connect_to_region(values.region)
+            s3conn = None
```

Comment:
My linter complained :( It's not necessary but its apparently more correct.  Let me know if you'd like me to remove it.